client/src/index.css

```diff
@@ -12,3 +8,7 @@
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
+
+body {
```

Comment:
What you are doing here is called "css reset" or "css normalize".
No changed required but you can checkout these sites:
https://meyerweb.com/eric/tools/css/reset/
https://necolas.github.io/normalize.css/